J/JuliaInterpreter/Compat.toml

```diff
@@ -1,20 +1,39 @@
-["0-0.7"]
-julia = "1"
+["0 - 0.7"]
+julia = "1.0"
+# assuming exact match was wanted
+# otherwise, use ~ or ^ specifier
```

Comment:
For the same reasons, remove all comments.  The script that automatically generates this file doesn't know anything about comments.
```suggestion
```